app/models/user.rb

```diff
@@ -22,4 +24,16 @@ def ensure_an_admin_remains
         raise Error.new "Can't delete last user"
       end
     end
+
+    def email_verification_check_for_destroy
+      if email == 'admin@depot.com'
+        raise Error.new "Can't delete the Admin"
+      end
+    end
+
+    def email_verification_check_for_update
+      if email == 'admin@depot.com'
```

Comment:
@avneetsng Extract this in private method and use the method in inline callback conditions.